spec/factories/payments.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :payment do
+    sender { nil }
+    receiver { nil }
```

Comment:
Creo que sender y receiver no deberian ser `nil`. Rails desde su version 5 hace que las [relaciones sean `required`por defecto](https://blog.bigbinary.com/2016/02/15/rails-5-makes-belong-to-association-required-by-default.html), esto significa que, a menos que indiques lo contrario, un `Payment` no puede existir sin `sender` o `receiver`. Para crear al usuario `sender` y `receiver` en esta factory, puedes ocupar la factory de `user`. [En este link se ve un ejemplo](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations), en el ejemplo un `Post` tiene un `author`, que es un `user` por lo tanto ocupa la `factory: user`.